### PR TITLE
Challenge - ES6: Use getters and setters to Control Access to an Object - Added solution and explanation of private variable naming convention

### LINKED ISSUE #18122: Add _underscore explanation to lesson
#### Describe your problem and - if possible - how to reproduce it
In the code snippet below, I was unaware of the purpose of the underscores. On stackoverflow, I learned that they are naming conventions used to signify private/protected variables and that they don't actually make the variable private. I think an explanation of this convention should be included inside of this lesson. 

```
class Book {
  constructor(author) {
    this._author = author;
  }
  // getter
  get writer(){
    return this._author;
  }
  // setter
  set writer(updatedAuthor){
    this._author = updatedAuthor;
  }
}
const lol = new Book('anonymous');
console.log(lol.writer);  // anonymous
lol.writer = 'wut';
console.log(lol.writer);  // wut
```

#### Add a Link to the page with the problem

https://learn.freecodecamp.org/javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object/

### PR DESCRIPTION
**Pull request by alanpaulprice**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x ] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [x ] None of my changes are plagiarized from another source without proper attribution.
- [x ] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #18122

### CODE REVIEW

**Review by ezioda004 — COMMENTED** [review]
Perhaps adding a "Note:" in a new line followed by the text would be better.

**Comment by alanpaulprice** [comment]
@ezioda004 Sure thing! Just to clarify, are you suggesting that the `Note:` should share a line with the text, like this?

`Note: It is convention to precede the name of a private variable with an underscore (<code>_</code>). The practice itself does not make a variable private.`

**Comment by ezioda004** [comment]
@alanpaulprice Yes, like that.

**Comment by ezioda004** [comment]
Hey @alanpaulprice,
Can you update to add the note like a heading. I think that'd look far better. Example:

#### Note
It is convention to precede the name of a private variable with an underscore (<code>_</code>). The practice itself does not make a variable private.

**Comment by alanpaulprice** [comment]
@ezioda004 Sure thing!

**Comment by ezioda004** [comment]
@alanpaulprice In view mode it's showing in the same line, can you check and add line break?

**Comment by alanpaulprice** [comment]
@ezioda004 Looks like my 2 latest commits on this branch have failed the Travis checks. I can't see any issues with the code.

Are there still problems with the checks?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ You can obtain values from an object, and set a value of a property within an ob
 These are classically called <dfn>getters</dfn> and <dfn>setters</dfn>.
 Getter functions are meant to simply return (get) the value of an object's private variable to the user without the user directly accessing the private variable.
 Setter functions are meant to modify (set) the value of an object's private variable based on the value passed into the setter function. This change could involve calculations, or even overwriting the previous value completely.
+<strong>Note</strong><br>It is convention to precede the name of a private variable with an underscore (<code>_</code>). The practice itself does not make a variable private.
```
Comment (ezioda004) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-getters-and-setters-to-control-access-to-an-object.english.md]: If you check in the view mode here, **Note** is in the same line as previous sentence, You can add a `<br>` after previous sentence.

**Comment by ezioda004** [comment]
@alanpaulprice Dont worry about that, the issue is from our side and will be fixed soon.
When the travis build is fixed I'll manually restart the check for this PR.

**Review by ezioda004 — APPROVED** [review]

**Comment by ezioda004** [comment]
Thank you @alanpaulprice for your contribution and congratulations on your first PR merge to this repo! 🎉
Hope to see more contributions from you in the future.

**Pull request merged**